packages/components/src/icons/angle-double-up-regular.svg

```diff
@@ -1 +1,3 @@
-<svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="angle-double-up" class="svg-inline--fa fa-angle-double-up fa-w-10" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="currentColor" d="M168.5 84.2l148 146.8c4.7 4.7 4.7 12.3 0 17l-19.8 19.8c-4.7 4.7-12.3 4.7-17 0L160 149.3 40.3 267.8c-4.7 4.7-12.3 4.7-17 0L3.5 248c-4.7-4.7-4.7-12.3 0-17l148-146.8c4.7-4.7 12.3-4.7 17 0zm-17 160L3.5 391c-4.7 4.7-4.7 12.3 0 17l19.8 19.8c4.7 4.7 12.3 4.7 17 0L160 309.3l119.7 118.5c4.7 4.7 12.3 4.7 17 0l19.8-19.8c4.7-4.7 4.7-12.3 0-17l-148-146.8c-4.7-4.7-12.3-4.7-17 0z"></path></svg>
\ No newline at end of file
+<svg aria-hidden="true" data-prefix="far" data-icon="angle-double-up" class="angle-double-up-regular_svg__angle-double-up-regular_svg__svg-inline--fa angle-double-up-regular_svg__angle-double-up-regular_svg__fa-angle-double-up angle-double-up-regular_svg__angle-double-up-regular_svg__fa-w-10" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
```

Comment:
@nawatts Ugh, when you run the command it is making class names longer, and also changing IDs (it changed the one on the loading spinner, which caused a failure). I'll have to shelve this for now and figure it out later.

Reply:
It looks like the IDs can be preserved by configuring one of the plugins.
https://svgo.dev/docs/plugins/cleanup-ids/

It may be worth adding the SVGO configuration to build-utils.